pkg/cmd/server/server.go

```diff
@@ -108,18 +108,19 @@ type Config struct {
 	ClusterDispatchCacheConfig CacheConfig `debugmap:"visible"`
 
 	// API Behavior
-	DisableV1SchemaAPI                bool          `debugmap:"visible"`
-	V1SchemaAdditiveOnly              bool          `debugmap:"visible"`
-	MaximumUpdatesPerWrite            uint16        `debugmap:"visible"`
-	MaximumPreconditionCount          uint16        `debugmap:"visible"`
-	MaxDatastoreReadPageSize          uint64        `debugmap:"visible"`
-	StreamingAPITimeout               time.Duration `debugmap:"visible"`
-	WatchHeartbeat                    time.Duration `debugmap:"visible"`
-	MaxReadRelationshipsLimit         uint32        `debugmap:"visible"`
-	MaxDeleteRelationshipsLimit       uint32        `debugmap:"visible"`
-	MaxLookupResourcesLimit           uint32        `debugmap:"visible"`
-	MaxBulkExportRelationshipsLimit   uint32        `debugmap:"visible"`
-	EnableExperimentalLookupResources bool          `debugmap:"visible"`
+	DisableV1SchemaAPI                       bool          `debugmap:"visible"`
+	V1SchemaAdditiveOnly                     bool          `debugmap:"visible"`
+	MaximumUpdatesPerWrite                   uint16        `debugmap:"visible"`
+	MaximumPreconditionCount                 uint16        `debugmap:"visible"`
+	MaxDatastoreReadPageSize                 uint64        `debugmap:"visible"`
+	StreamingAPITimeout                      time.Duration `debugmap:"visible"`
+	WatchHeartbeat                           time.Duration `debugmap:"visible"`
+	MaxReadRelationshipsLimit                uint32        `debugmap:"visible"`
+	MaxDeleteRelationshipsLimit              uint32        `debugmap:"visible"`
+	MaxLookupResourcesLimit                  uint32        `debugmap:"visible"`
+	MaxBulkExportRelationshipsLimit          uint32        `debugmap:"visible"`
+	EnableExperimentalLookupResources        bool          `debugmap:"visible"`
+	EnableExperimentalRelationshipExpiration bool          `debugmap:"visible"`
```

Comment:
please add default annotation

Reply:
Its not defaulted here - we default it on the flag